e2e/sdk_remote_integration.spec.ts

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: Apache-2.0
+"use strict";
+
+import { TxReceipt } from "#erdstall/api/responses";
+import { Mint } from "#erdstall/api/transactions";
+import { Address } from "#erdstall/ledger";
+import { Assets } from "#erdstall/ledger/assets";
+import * as assets from "#erdstall/ledger/assets";
+import * as sdk from "#erdstall";
+import { Erdstall } from "#erdstall";
+
+import { ethers } from "ethers";
+import * as fs from "fs";
+import { exec } from "child_process";
+
+(global as any).WebSocket = require("ws");
```

Comment:
Not that I know of. This was the recommended go-to solution, any way.

Reply:
No, we were crying.
Imported a bunch of libs to no avail. This was the only way, even with `jsdom-global` as an extension to mocha, there is no default `Websocket` available.

Although one solution would be to always depend on a websocket library in all our code. Then this would be superfluous, but then again... Why should we be forced to do this?

Reply:
OK I see, reality strikes again. Then it's fine 🤡